numba/targets/codegen.py

```diff
@@ -387,9 +389,63 @@ def get_pointer_to_function(self, name):
         return self._codegen._engine.get_function_address(name)
 
     def _finalize_specific(self):
+        self._codegen._scan_and_fix_unresolved_refs(self._final_module)
         self._codegen._engine.finalize_object()
 
 
+class RuntimeLinker(object):
+    """
+    For tracking unresolved symbols generated at runtime due to recursion.
+    """
+    PREFIX = '.numba.unresolved$'
```

Comment:
Is this defined elsewhere too? Was wondering if the place where the symbols with this prefix are generated and the above definition ought to be in the same place?